### PR TITLE
Add copyright notice

### PR DESCRIPTION
**Pull request by dannylamb**
**GitHub Issue**: https://github.com/Islandora/documentation/issues/1267

# What does this Pull Request do?

Adds an COPYRIGHT file to go along with our GPL2 LICENSE file.  We've supposed to have been doing this since the beginning to comply with GPL2, so here it is.  The copyright assertion itself matches that of our MIT licensed offerings.

# Interested parties
@Islandora/8-x-committers @manez

### CODE REVIEW

**Comment by whikloj** [comment]
@dannylamb I've noticed [this error](https://travis-ci.com/Islandora/controlled_access_terms/jobs/273651714#L1167) before, do you want to fix it here to get Travis to pass? It looks like either you can keep it as an array and make it `$delimiters` or make it `private const DELIMITERS`

**Comment by dannylamb** [comment]
I'll take a look :man_mechanic:

File: src/Plugin/Field/FieldFormatter/EDTFFormatter.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class EDTFFormatter extends FormatterBase {
    *
    * @var array
    */
-  private $DELIMITERS = [
+  private const $DELIMITERS = [
```
Comment (seth-shaw-unlv) [inline src/Plugin/Field/FieldFormatter/EDTFFormatter.php]: Remove the `$`. PHP constants don't use the dollar-sign signifier. It results in an error: `ParseError: syntax error, unexpected '$DELIMITERS' (T_VARIABLE)`

Comment (dannylamb) [reply src/Plugin/Field/FieldFormatter/EDTFFormatter.php]: :man_facepalming:

**Pull request merged**